### PR TITLE
Ported to Hapi v17

### PR DESCRIPTION
**Pull request by jlopezxs**
Support to Hapi v17 #3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@softonic/hapi-access-logger",
-  "version": "2.1.0",
+  "version": "3.0.0",
```
Comment (rubennorte) [inline package.json]: Version should be updated using the `release` script.

File: package.json [hunk 2]
```diff
@@ -35,15 +35,15 @@
...
-    "hapi": "^16.1.0",
-    "jest": "^19.0.2"
+    "babel-cli": "^6.26.0",
+    "babel-preset-latest": "^6.24.1",
+    "babel-register": "^6.26.0",
+    "eslint": "^4.10.0",
+    "eslint-config-airbnb-base": "^12.1.0",
+    "eslint-plugin-import": "^2.8.0",
+    "eslint-plugin-jest": "^21.2.0",
+    "hapi": "^17.0.1",
```
Comment (rubennorte) [inline package.json:45]: I think we should start using `peerDependencies` in plugins. In this case `"hapi": "17.x.x"`

**Review by rubennorte — COMMENTED** [review]
I think it'd be a good idea to use `babel-preset-env` with Node 8 (and add it to `engines` in the `package.json`). Hapi v17 only works with Node 8 either way, so we make it clearer that this plugin requires that version.

File: package.json (hunk 2 above)
Comment (jlopezxs) [reply package.json:45]: Solved in #5

**Pull request merged**